test/martha/martha_v3.test.js

```diff
@@ -933,7 +872,9 @@ test.serial('martha_v3 should return 500 if access method parsing fails', async
     );
 });
 
-test.serial('martha_v3 should return 500 on exception trying to get access token from Bond', async (t) => {
+/* BT-236 Skip testing access token fetch failure since that is not something this code even attempts with BDC
+ * signed URLs turned off. */
+test.skip('martha_v3 should return 500 on exception trying to get access token from Bond', async (t) => {
```

Comment:
I don't like leaving commented out code like this (search `ignore` in the Cromwell repo) but hopefully we'll be back soon 🤞

Reply:
yeah I'm not imagining this as a "TODO" that never gets "TODONE" but as a PR that will be completely reverted in a week or two.